charts/fluid-datamigrate/juicefs/templates/statefulset.yaml

```diff
@@ -64,16 +65,16 @@ spec:
             - mountPath: /etc/fluid/scripts
               name: data-migrate-script
       affinity:
-        # prefer to use different nodes to maximize bandwidth
+        # prefer to use different nodes to maximize bandwidth, the launcher is also a worker.
         podAntiAffinity:
           preferredDuringSchedulingIgnoredDuringExecution:
             - podAffinityTerm:
                 labelSelector:
                   matchExpressions:
-                    - key: app
+                    - key: fluid.io/operation
                       operator: In
                       values:
-                        - {{ printf "%s-workers" .Release.Name }}
+                        - migrate-{{ .Release.Namespace }}-{{ .Release.Name }}
```

Comment:
Why don't we reuse `{{ .Values.datamigrate.affinity }}` just like what we do in job.yaml?

Reply:
fixed.